src/confcom/azext_confcom/tests/latest/README.md

```diff
@@ -167,4 +167,9 @@ test_fragment_incorrect_sidecar | mcr.microsoft.com/aci/msi-atlas-adapter:master
 test_signing | mcr.microsoft.com/acc/samples/aci/helloworld:2.8 | Sign a fragment with a key and chain file
 test_generate_import | mcr.microsoft.com/acc/samples/aci/helloworld:2.8 | Generate an import statement for the signed fragment file
 test_local_fragment_references | mcr.microsoft.com/acc/samples/aci/helloworld:2.8 | Make sure the fragment references are correct when the fragment is local
+test_registry_is_running | N/A | See if the local registry is running
+test_generate_import_from_remote | mcr.microsoft.com/acc/samples/aci/helloworld:2.8 | Test generating an import statement from the feed of a signed standalone fragment
```

Comment:
This is on version 2.9 now

Reply:
The image itself doesn't really matter as long as it's in MCR. Do you think we should update all of the 2.8s to 2.9s?

Reply:
I'm not sure if you'd get flagged in S360 for using old images in the pipeline. Otherwise it wouldn't matter. Also, I'll have to make a 2.10 soon anyway.

Reply:
I don't think it would flag? We can make a PR for 2.10 once it's ready